src/pages/Main/BestSeller/BestSeller.js

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import './BestSeller.scss';
+import BestSellerCard from './BestSellerCard/BestSellerCard';
+
+function BestSeller() {
+  const [currentPosition, setCurrentPosition] = useState(0);
+  const [transitionTime, setTransitionTime] = useState(0.3);
+  const cardSize = 23;
+
+  const handleClickLeftButton = () => {
+    if (currentPosition === 0) {
+      setTransitionTime(0);
+      setCurrentPosition(CardsList.length - 1 + 1);
+      setTimeout(() => {
+        setTransitionTime(0.3);
+        setCurrentPosition(CardsList.length - 1);
+      }, 10);
+    } else {
+      setCurrentPosition(prev => prev - 1);
+    }
+  };
+  const handleClickRightButton = () => {
+    if (currentPosition === CardsList.length - 1) {
+      setTransitionTime(0);
+      setCurrentPosition(-1);
+      setTimeout(() => {
+        setTransitionTime(0.3);
+        setCurrentPosition(0);
+      }, 0);
+    } else {
+      setCurrentPosition(prev => prev + 1);
+    }
+  };
+
+  return (
+    <div className="bestSeller">
+      <div className="header">
+        <h1 className="title">Best Seller</h1>
+        <button className="linkButton">
+          더 많은 제품 보기&nbsp;&nbsp;&nbsp;>
+        </button>
+      </div>
+      <div
+        className="slide"
+        style={{
+          transform: `translateX(${(currentPosition - 1) * -cardSize}rem)`,
+          transition: `all ${transitionTime}s`,
+        }}
+      >
+        {dummyCardsList.map((card, idx) => {
+          const hasTwoCard = card.imgUrl.length === 2;
+          return (
+            <BestSellerCard
+              key={idx}
+              hasTwoCard={hasTwoCard}
+              imgUrl={card.imgUrl}
+            />
+          );
+        })}

```

Comment:
매개변수 자리에서도 구조분해 가능합니다.
때에 따라서는 조금 더 가독성이 높아질 수 있어요!
```suggestion
{dummyCardsList.map(({ imgUrl }, idx) => {
          const hasTwoCard = imgUrl.length === 2;
          return (
            <BestSellerCard key={idx} hasTwoCard={hasTwoCard} imgUrl={imgUrl} />
          );
        })}

```